src/main/java/com/redhat/pantheon/servlet/ModuleRevisionUpload.java

```diff
@@ -54,11 +56,15 @@
 
     private static final Logger log = LoggerFactory.getLogger(ModuleRevisionUpload.class);
 
+    private AsciidoctorService asciidoctorService;
+
     private AsciidoctorPool asciidoctorPool;
 
     @Activate
-    public ModuleRevisionUpload(@Reference AsciidoctorPool asciidoctorPool) {
+    public ModuleRevisionUpload(@Reference AsciidoctorPool asciidoctorPool,
+                                @Reference AsciidoctorService asciidoctorService) {
         this.asciidoctorPool = asciidoctorPool;
```

Comment:
I wonder if we still need the pool object here. Since we are calling the `asciidoctorService` directly now, I don't think we need to get asciidoctor instances directly anymore.